File: C# .Net/JDI UI Framework/Tests/Scenarios/Entities/Attendee.cs
namespace Epam.Tests.Scenarios.Entities
{
    public class Attendee
    {
        public JobSearchFilter Filter = new JobSearchFilter();

        public string Name = "Roman";
        public string LastName = "Iovlev";
        public string Email = "roman_iovlev@epam.com";
        public string Country;
        public string City = "Saint-Petersburg";
        public string Cv;// = "jdi-uitest-tutorialtests\\src\\test\\resources\\cv.txt";
        public string Comment = "I WANT TO WORK IN EPAM!!!";
        
        public new string ToString()
        {
            return Name + " " + LastName;
        }
    }
}

File: Java/Tests/jdi-uitest-tutorialtests/src/main/java/com/epam/jdi/entities/Attendee.java
package com.epam.jdi.entities;


import com.epam.commons.DataClass;

/**
 * Created by Roman_Iovlev on 10/22/2015.
 */
/*@AllArgsConstructor
@NoArgsConstructor*/
public class Attendee extends DataClass {
    public JobSearchFilter filter = new JobSearchFilter();

    public String name = "Roman";
    public String lastName = "Iovlev";
    public String email = "roman_iovlev@epam.com";
    public String country = "Russian Federation";
    public String city = "Saint-Petersburg";
    public String cv; // = "jdi-uitest-tutorialtests\\src\\test\\resources\\cv.txt";
    public String comment = "I WANT TO WORK IN EPAM!!!";

    public Attendee(String name) {
        this.name = name;
    }

    public Attendee(){
    }
}
